website/docs/md_components/inpage_navigation_components/usage_markdown.md

```diff
@@ -1 +1,18 @@
-No specific UX guidelines is missing
\ No newline at end of file
+A Back to Top button, which is a shortcut that allows users to quickly navigate to the top of the page.
+
+When users get at the bottom of a really long page, they often will need to get back to the top to:
+
+- See the navigation menu and choose a new destination
+- Access filtering and sorting features
+- Enter or edit a query in the search box
+
+When the page is really long (for example, because of infinite scrolling), it becomes too tedious to scroll back up through screens and screens of content. Hence the role of the Back to Top button.
+
+### Do's
+- Use a Back to Top button for pages that are longer than 4 screens. For relatively short page lengths, Back to Top links are overkill 

```

Comment:
Use backtics for ` ` for special identifiers such as any type of elements or components.
So I would wrap `Back to Top` with a backtics everywhere in this file.